src/test/deserialization_tests.cpp

```diff
@@ -120,125 +120,125 @@ class GRPCPredictRequest_2 : public TensorflowGRPCPredict_2 {
+    InputSink<ov::runtime::InferRequest&> inputSink(inferRequest);
     Status status;
-    status = deserializePredictRequest_2<MockTensorProtoDeserializator_2>(
+    status = deserializePredictRequest<MockTensorProtoDeserializator>(
         request, tensorMap, inputSink, isPipeline);
     EXPECT_EQ(status, ovms::StatusCode::OV_INTERNAL_DESERIALIZATION_ERROR) << status.string();
 }
 
-TEST_F(GRPCPredictRequest_2, ShouldSuccessForSupportedPrecision) {
+TEST_F(GRPCPredictRequest, ShouldSuccessForSupportedPrecision) {
     ov::runtime::Core ieCore;
     std::shared_ptr<ov::Model> network = ieCore.read_model(std::filesystem::current_path().u8string() + "/src/test/dummy/1/dummy.xml");
     ov::runtime::CompiledModel execNetwork = ieCore.compile_model(network, "CPU");
     ov::runtime::InferRequest inferRequest = execNetwork.create_infer_request();
-    InputSink_2<ov::runtime::InferRequest&> inputSink(inferRequest);
-    auto status = deserializePredictRequest_2<ConcreteTensorProtoDeserializator_2>(request, tensorMap, inputSink, isPipeline);
+    InputSink<ov::runtime::InferRequest&> inputSink(inferRequest);
+    auto status = deserializePredictRequest<ConcreteTensorProtoDeserializator>(request, tensorMap, inputSink, isPipeline);
     EXPECT_TRUE(status.ok());
 }
 
-TEST_P(DeserializeTFTensorProtoNegative_2, ShouldReturnNullptrForPrecision) {
+TEST_P(DeserializeTFTensorProtoNegative, ShouldReturnNullptrForPrecision) {
     ovms::Precision testedPrecision = GetParam();
     tensorMap[tensorName]->setPrecision(testedPrecision);
-    ov::runtime::Tensor tensor = deserializeTensorProto_2<ConcreteTensorProtoDeserializator_2>(tensorProto, tensorMap[tensorName]);
+    ov::runtime::Tensor tensor = deserializeTensorProto<ConcreteTensorProtoDeserializator>(tensorProto, tensorMap[tensorName]);
     EXPECT_FALSE((bool)tensor) << "Unsupported OVMS precision:"
                                << toString(testedPrecision)
                                << " should return nullptr";
 }
 
-TEST_P(DeserializeTFTensorProto_2, ShouldReturnValidBlob) {
+TEST_P(DeserializeTFTensorProto, ShouldReturnValidBlob) {
     ovms::Precision testedPrecision = GetParam();
     SetUpTensorProto(TensorInfo::getPrecisionAsDataType(testedPrecision));
     tensorMap[tensorName]->setPrecision(testedPrecision);
-    ov::runtime::Tensor tensor = deserializeTensorProto_2<ConcreteTensorProtoDeserializator_2>(tensorProto, tensorMap[tensorName]);
+    ov::runtime::Tensor tensor = deserializeTensorProto<ConcreteTensorProtoDeserializator>(tensorProto, tensorMap[tensorName]);
     EXPECT_TRUE((bool)tensor) << "Supported OVMS precision:"
                               << toString(testedPrecision)
                               << " should return valid blob ptr";
 }
 
 INSTANTIATE_TEST_SUITE_P(
     TestDeserialize,
-    GRPCPredictRequestNegative_2,
+    GRPCPredictRequestNegative,
     ::testing::ValuesIn(UNSUPPORTED_INPUT_PRECISIONS_2),
```

Comment:
Is it intentional that `UNSUPPORTED_INPUT_PRECISIONS_2` and `SUPPORTED_INPUT_PRECISIONS_2` names end with `_2`?

Reply:
Good catch